shared/constants/entities.js

```diff
@@ -27,44 +27,64 @@ export type Actions = Delete | Merge | Replace
 
 // State
 export type State = KBRecord<{
-  devices: Map<string, DeviceDetailRecord>,
-  teams: Teams.TeamRecord,
-  searchResults: Map<SearchConstants.SearchResultId, KBRecord<SearchConstants.SearchResult>>,
-  searchQueryToResult: Map<SearchConstants.SearchQuery, List<SearchConstants.SearchResultId>>,
-  messages: Map<ChatConstants.MessageKey, ChatConstants.Message>,
+  attachmentDownloadProgress: Map<ChatConstants.MessageKey, ?number>,
```

Comment:
sorted + new inbox keys. we could make this more hierarchical but leaving that for now